tools/nvcc_fix_deps.py

```diff
@@ -91,11 +94,22 @@ def extract_one(name: str, i: int, args: List[str]) -> Optional[str]:
         if path is not None:
             include_dirs.append(Path(path).resolve())
             return
-
+        
 
 if __name__ == "__main__":
+    
+    # Escape the command to prevent command injection
+    cmd = 'nvcc ' + sys.argv[1:]
+    if os.name == 'nt':
```

Comment:
Hmm, is `cygwin` considered posix?

Reply:
Yes, and I believe this still fill fixes it because it'll take shell quotation. https://stackoverflow.com/questions/32612235/what-is-os-name-on-cygwin